### PR TITLE
Add prometheus cloudwatch exporter package.

### PR DESCRIPTION
**Pull request by dlorenc**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [X] REQUIRED - The package is added to `packages.txt`

### CODE REVIEW

File: cloudwatch-exporter.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  version: "0.15.3"
+  epoch: 0
+  description: Metrics exporter for Amazon AWS CloudWatch
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation:
+      license: Apache-2.0
```
Comment (imjasonh) [inline cloudwatch-exporter.yaml:12]:
```suggestion
    - license: Apache-2.0
```

File: cloudwatch-exporter.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package:
+  name: cloudwatch-exporter
+  version: "0.15.3"
+  epoch: 0
+  description: Metrics exporter for Amazon AWS CloudWatch
+  target-architecture:
+    - all
```
Comment (imjasonh) [inline cloudwatch-exporter.yaml:7]:
```suggestion
```

File: cloudwatch-exporter.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  - uses: git-checkout
+    with:
+      repository: https://github.com/prometheus/cloudwatch_exporter
+      tag: v${{package.version}}
+      expected-commit: eb9f513167eb1bf2efeadbf67e9641b610d3dd4b
+
+  - runs: |
+      # CHECK-WHEN-VERSION-CHANGES: 0.15.3
+      # Address CVE-2022-1471
+      sed -i 's/1.33/2.0/g' pom.xml
```
Comment (kaniini) [inline cloudwatch-exporter.yaml:36]: humm, this seems very fragile.  should we maybe have a tool which actually understands the XML and does the edits?

Comment (dlorenc) [reply cloudwatch-exporter.yaml:36]: Swapped it to just applying a patch.

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**